packages/swap/src/event-handlers/v140/insufficientBoostLiquidity.ts

```diff
@@ -1,17 +1,24 @@
 import { z } from 'zod';
 import { assetConstants } from '@/shared/enums';
-import { internalAssetEnum, u128 } from '@/shared/parsers';
+import { internalAssetEnum, rustEnum, u128 } from '@/shared/parsers';
 import { EventHandlerArgs } from '..';
 
 export const insufficientBoostLiquiditySchema = z.object({
   prewitnessedDepositId: u128,
   asset: internalAssetEnum,
   amountAttempted: u128,
   channelId: u128,
+  originType: rustEnum(['Vault', 'DepositChannel']).optional(),
 });
-
 export const insufficientBoostLiquidity = async ({ prisma, event, block }: EventHandlerArgs) => {
-  const { channelId, asset, amountAttempted } = insufficientBoostLiquiditySchema.parse(event.args);
+  const { channelId, asset, amountAttempted, originType } = insufficientBoostLiquiditySchema.parse(
+    event.args,
+  );
+
+  // no deposit channel to look up failed boosts for vault swaps
```

Comment:
don't  we want to make the deposit channel for boosts optional then? so we can capture this, or is that coming later?

Reply:
you are right, this would make sense - will change!